### PR TITLE
Final update to certbot-auto

### PR DESCRIPTION
**Pull request by bmw**
Fixes https://github.com/certbot/certbot/issues/8690.

You can see certbot-auto test farm tests passing with this change at https://dev.azure.com/certbot/certbot/_build/results?buildId=3515&view=results.

After this PR, we'll let the release script make its automated changes to certbot-auto as part of the 1.14.0 release and then never make any code changes to certbot-auto ever again!

### CODE REVIEW

**Review by ohemorange — COMMENTED** [review]
YEAHHHHHHH!!!!!

approved, but putting this as a comment because we should probably get a second review per our (SOON TO BE DEPRECATED) certbot-auto update policy

**Review by adferrand — APPROVED** [review]
And... that's it.

**Pull request merged**
